sssd_test_framework/hosts/keycloak.py

```diff
@@ -0,0 +1,62 @@
+"""Keycloak multihost host."""
+
+from __future__ import annotations
+
+from .base import BaseDomainHost
+
+__all__ = [
+    "KeycloakHost",
+]
+
+
+class KeycloakHost(BaseDomainHost):
+    """
+    Keycloak host object.
+
+    Provides features specific for Keycloak server.
+    """
```

Comment:
I know the docs hasn't been updated, but please put your sample code, and usage here.

Reply:
updating.